src/pages/login/index.js

```diff
@@ -16,6 +18,16 @@ const Login = () => {
 		setFormData({ ...formData, [name]: value });
 	};
 
+	const handleLogin = () => {
+		const attemptLogin = async () => {
+			const res = await onLogin(formData.email, formData.password)
+			res.status === 'fail' ? setError(true) : setError(false)
+		}
+		attemptLogin()
```

Comment:
If you make `handleLogin` an async function, you will avoid needing to declare and immediately use a one-use function.

Reply:
Donezel